### DISCUSSION TITLE
Google Analytics implementation

### DISCUSSION THREAD

**Discussion by sid-r-singh** [carbon-design-system/carbon#8676] [Ideas]
I am using Carbon-gatsby. How do I integrate Google Analytics or any other analytics for that matter ?

**Comment by joshblack** [comment]
Hi there @sid-r-singh! 👋 

Since this is a Gatsby Theme question, it might be best to ask over on: https://github.com/carbon-design-system/gatsby-theme-carbon

This repo/project is used for tracking the Design System itself which is why this question may not have received a response quickly (sorry about that!)

Hope this helps!

**Comment by VedatOlmez** [comment]
+
